tests/util/setup-dev-tests.ts

```diff
@@ -43,11 +43,8 @@ export interface DevTestContext {
 }
 
 interface InternalDevTestContext extends DevTestContext {
-  polkadotWsProviders: WsProvider[];
-  web3Providers: HttpProvider[];
-
-  // Internal member to keep track of web3 singleton
-  _polkadotApi: EnhancedWeb3;
+  _polkadotApis: ApiPromise[];
+  _web3Providers: HttpProvider[];
```

Comment:
why the underscore?

Reply:
Those are only for internal setup, not exposed. I tend to add the _ to ensure someone inspecting the object knows they are "private"